app/move/controllers/update/move-details.test.js

```diff
@@ -0,0 +1,166 @@
+const CreateMoveDetails = require('../create/move-details')
+const MixinProto = CreateMoveDetails.prototype
+const UpdateBaseController = require('./base')
+
+const moveService = require('../../../../common/services/move')
+
+// TODO: figure out how to proxyquire lodash without breaking it elsewhere
```

Comment:
I think you are thinking of `require('proxyquire').noPreserveCache();`

Reply:
👍